apiserver/controllers/controllers.go

```diff
@@ -180,6 +232,19 @@ func (a *APIController) NotFoundHandler(w http.ResponseWriter, r *http.Request)
 	json.NewEncoder(w).Encode(apiErr)
 }
 
+func (a *APIController) MetricsTokenHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	token, err := a.auth.GetJWTMetricsToken(ctx)
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	fmt.Fprintf(w, `{"token": "%s"}`, token)
```

Comment:
There is a ```JWTResponse{}``` in params. Please use that.

```suggestion
	json.NewEncoder(w).Encode(runnerParams.JWTResponse{Token: token})
```